modules/spring-config/src/main/java/org/mule/runtime/config/spring/dsl/model/ApplicationModel.java

```diff
@@ -779,6 +801,20 @@ private ComponentModel createOperationInstance(ComponentModel operationRefModel,
     return processorChainModel;
   }
 
+  /**
+   * visits all the possible types of a given parameter to realize if it's a "void return type", in which case the
+   * expanded chain will not modify the structure of the event
+   */
+  private class ReturnsVoidTypeVisitor extends MetadataTypeVisitor {
```

Comment:
I can't, as `IntrospectionUtils` lives in mule-extension-spring-support, which relies in mule-spring-config that's holding the ApplicationModel class